gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_5_0/02_add_shared_policy_group.yml

```diff
@@ -69,6 +69,10 @@ databaseChangeLog:
               - column: {name: deployed_at, type: timestamp(6), constraints: { nullable: true } }
```

Comment:
`addPrimaryKey` detected in liquibase changelog 💥. Please create the primary key directly with the create table statement.